packages/appeals-service-api/src/db/seed/data-dev.js

```diff
@@ -70,6 +70,7 @@ const appealCases = [
 		appealTypeName: 'Householder',
 		decision: 'refused',
 		originalCaseDecisionDate: pickRandom(pastDates),
+		outcome: 'allowed',
```

Comment:
Might be good to have an appeal with each outcome we expect? Will help the testers.

Reply:
Yes, I think Tom is working on something like this.